mods/modloader.json

```diff
@@ -0,0 +1,322 @@
+{
+    "format": 1,
+    "name": "Modloader",
+    "authors": ["Risugami"],
+    "versions": [
+        {
+            "name": "Modloader",
+            "mcvsn": ["1.2.4"],
+            "files": [
+                {
+                    "filename": "ModLoader 1.2.4.zip",
+                    "archived": "0F9BB79149F95061BEE539A3120D820043E84ED8747E9CA98756FC34E40B7D7D/Modloader 1.2.4.zip",
```

Comment:
Don't include these in your pull request. I have a script that will add them when I upload the files